server/services/reportingService.test.js

```diff
@@ -68,7 +69,39 @@ Bella,10
 Charlotte,5
 `)
 
-    expect(reportingClient.getMostOftenInvolvedPrisoners).toBeCalledWith('LEI', startDate, endDate)
+    expect(reportingClient.getMostOftenInvolvedPrisoners).toBeCalledWith('LEI', [startDate, endDate])
     expect(offenderService.getOffenderNames).toBeCalledWith('token-1', ['AAAA', 'BBBB', 'CCCC'])
   })
+
+  it('getIncidentsOverview', async () => {
+    reportingClient.getIncidentsOverview.mockReturnValue([
+      {
+        total: 'total-1',
```

Comment:
It probably doesn't matter that you've used strings, but shouldn't each of the values be a number?

Reply:
yeah - it was just easier to provide a unique string than a unique number.. being a bit lazy. I can correct..